### ISSUE TITLE
Updates/ features 

### ISSUE THREAD

**Issue by faierbol** [marketplacekit/marketplacekit#58]
Are there any updates/ features planned for this app? :)

**Comment by marketplacekit** [comment]
Yes there will be, we're finishing off the mobile API and adding some functionality to add other payment gateways, once that's done we'll update the repo.

**Comment by faierbol** [comment]
Happy New Year!!!

**Comment by ovresko** [comment]
app looks dead after 6 months since last commit! is it?

**Comment by faierbol** [comment]
It seems like it is :(

**Comment by marketplacekit** [comment]
sorry for the lack of updates guys. it's not abandoned, no.

we've been busy with client work and the hosted version for a while. Few more weeks and we'll start making updates.

**Comment by igibioy999** [comment]
It would be great to see the adding of payment gateways (like in the hosted version) as well as some of the addons such as the Translation module published. Appreciate your effort on this!

**Comment by faierbol** [comment]
@marketplacekit can you push the web version on github and also the addons?

**Comment by faierbol** [comment]
7 months past away and no updates :(
